Modules/Messages/test/Messages.Application.Tests/Messages.Application.Tests.csproj

```diff
@@ -14,7 +16,9 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<ProjectReference Include="..\..\src\Messages.Application\Messages.Application.csproj" />
+        <ProjectReference Include="..\..\src\Messages.Application\Messages.Application.csproj" />
+        <ProjectReference Include="..\..\src\Messages.Infrastructure\Messages.Infrastructure.csproj" />
+        <ProjectReference Include="..\..\..\Devices\test\Devices.Application.Tests\Devices.Application.Tests.csproj" />
```

Comment:
To get the `TestDataGenerator`, the one with `CreateRandomIdentityAddress()` and `CreateRandomDeviceId()`.

Reply:
One of the most important rules for the backbone architecture is: no references between modules!

If there are things that are required by multiple modules, you should move these things to the building blocks.

Reply:
Why did you resolve the comment even though you didn't fix it?

Reply:
Ok, didn't know that but it actually makes sense. I will refactor the methods from the TestDataGenerator in Devices.Application.Tests so that they are mostly in UnitTestTools.

I thought that explanation suffices to settle the matter, my bad - misunderstanding :)